packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -85,86 +84,122 @@ In the future when decorators become a native web platform feature, this may no
 
 ### Using decorators with TypeScript { #decorators-typescript }
 
-To use decorators with [TypeScript](https://www.typescriptlang.org/docs/handbook/decorators.html), enable the `experimentalDecorators` compiler option.
+TypeScript supports both experimental decorators and standard decorators. We recommend that TypeScript developers use experimental decorators for now for optimal compiler output.
+
+To use experimental decorators you must enable the `experimentalDecorators` compiler option.
 
-You should also ensure that the `useDefineForClassFields` setting is `false`. Note, this should only be required when the `target` is set to `esnext` or greater, but it's recommended to explicitly ensure this setting is `false`.
+You should also ensure that the `useDefineForClassFields` setting is `false`. Note, this is only be required when the `target` is set to `ES2022` or greater, but it is recommended to explicitly set this to `false`.
 
 ```json
-"experimentalDecorators": true,
-"useDefineForClassFields": false,
+// tsconfig.json
+{
+  "compilerOptions": {
+    "experimentalDecorators": true,
+    "useDefineForClassFields": false
+  }
+}
 ```
 
 Enabling `emitDecoratorMetadata` is not required and not recommended.
 
-### Using decorators with Babel  { #decorators-babel }
+#### Migrating TypeScript experimental decorators to standard decorators { #migrating-typescript-standard-decorators }
+
+Lit decorators have been designed to support standard decorator syntax (using `accessor` on class field decorators) with TypeScript's experimental decorator mode.
+
+This allows incremental migration off of experimental decorators by adding the `accessor` keyword to decorated properties without a change of behavior. Once all class field decorators use the `accessor` keyword, you can change your `tsconfig.json` completing the migration to standard decorators:
+
+```json
+// tsconfig.json
+{
+  "compilerOptions": {
+    "experimentalDecorators": false,
+    "useDefineForClassFields": true
```

Comment:
These are the default values, so I don't know if we need to show them.

Reply:
tsconfig stuff is kinda iffy because the default is different depending on other configs or TS version so I don't think it's bad to show them to be explicit but we could add `//` comment notation stating it could be omitted with certain TS versions.

Reply:
I like the comment approach.